commands/version/index.js

```diff
@@ -346,6 +377,7 @@ class VersionCommand extends Command {
           changelogPreset,
           rootPath,
           tagPrefix: this.tagPrefix,
+          prereleaseId: getPrereleaseId(node),
```

Comment:
Logic determining whether to prerelease stable packages (`shouldPrerelease`) or graduate prereleasePackages (`shouldGraduate`) is encapsulated here so that `ConventionalCommitUtilties.recommendVersion` is only reacting to the presence or absence of `prereleaseId`.